src/h5web/visualizations/heatmap/LogScaleToggler.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { FiEyeOff, FiEye } from 'react-icons/fi';
+import styles from './HeatmapVis.module.css';
+
+interface Props {
+  logScale: boolean;
+  setLogScale: React.Dispatch<React.SetStateAction<boolean>>;
+}
+
+function LogScaleToggler(props: Props): JSX.Element {
+  const { logScale, setLogScale } = props;
+
+  return (
+    <button
+      onClick={() => {
+        setLogScale(!logScale);
+      }}
+      type="button"
+      role="switch"
+      aria-checked={logScale}
+      className={styles.scaleTypeSelector}
+    >
+      {logScale ? <FiEye /> : <FiEyeOff />}
+      SymLog scale
+    </button>
+  );
+}
+
+export default LogScaleToggler;
```

Comment:
The logic in there is quite simple but I preferred to extract it as a component as the `HeatmapVis` starts to be a bit bloated...